### DISCUSSION TITLE
Do what thunderbird did, be more loud

### DISCUSSION THREAD

**Discussion by ronniedroid** [FossifyOrg/General-Discussion#20] [General]
Dear @naveensingh 

Hope you are doing well and having fun amidst all this pressure.

I was thinking we should do what Thunderbird did and be a lot louder, things I have in mind are

- Post a blog post about the development of the project ones every week or bimonthly.
- Setup a podcast if possible.
- Share more on social media (X, Reddit, Instagram etc..)
- Seek interviews by big (or big FOSS) YouTubers.

You could also apply for FUTO org's fund, to help kick-start it all.

**Comment by naveensingh** [comment]
@ronniedroid Great ideas. I like to keep a 'low profile' so I probably won't be seeking interviews. As long as I can help it, I don't want to be the center of attention nor do I want credits, that's why I created a separate organization instead of just maintaining the apps on my account. If the apps get popular, the organization will become a symbol on its own. It doesn't need a face.

To keep everyone updated, I will start posting on the blogs and socials later as I'm able to increase my time here. Right now the priority is release and maintenance.

**Reply by inson1** [reply]
@naveensingh With "face" you get more donates imo = more time to work on this project

**Reply by naveensingh** [reply]
> With "face" you get more donates

@inson1 You are not wrong. As I said in some other post, we'll do whatever it takes to keep things fossified.

**Comment by naveensingh** [comment]
> You could also apply for FUTO org's fund, to help kick-start it all.

I'll maybe try that, not sure if they accept projects still in their infancy.

**Reply by ronniedroid** [reply]
Yes please do that, they are all for freedom respecting software and I believe they will be able to help in a way.

**Reply by Secret-chest** [reply]
Doesn't @tibbi work at FUTO?

**Reply by ronniedroid** [reply]
Are you asking, or you do know something?

I don't think he doe, but even if he did, why would it matter?

**Reply by inson1** [reply]
@Secret-chest He workd at SMT. Idk where he is now.
